packages/gatsby/index.d.ts

```diff
@@ -11,7 +11,8 @@ import {
   ComposeScalarTypeConfig,
   ComposeUnionTypeConfig,
 } from "graphql-compose"
-import { GraphQLOutputType } from "graphql"
+import { GraphQLOutputType, GraphQLFieldConfigMap } from "graphql"
+import { PluginRef } from "./src/bootstrap/load-plugins/types"
```

Comment:
@antoinerousseau This `PluginRef` type import seems to cause some conflicts ( there is `PluginRef` type defined in this file already)